apps/rush-lib/src/scripts/install-run-rush.ts

```diff
@@ -49,13 +49,15 @@ function run(): void {
     ...packageBinArgs /* [build, --to, myproject] */
   ]: string[] = process.argv;
 
+  const scriptName = path.basename(scriptPath);
```

Comment:
Add type annotations to these two lines:
```suggestion
  const scriptName: string = path.basename(scriptPath);
```